jest.config.js

```diff
@@ -0,0 +1,189 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Mind if we nix these file comments? Jest configuration is relatively straightforward and self-descriptive. Their documentation website should be able to clarify any confusion.

Reply:
This was the file generated from `jest --init`, is there a reason to get rid of it?

Reply:
Honestly, reading a wall of comments to find the right place to change a config stresses me out. The jest documentation does a much better job at conveying the meaning and implications of each setting than a single ascii line.